cli/cmd/project/logs.go

```diff
@@ -0,0 +1,109 @@
+package project
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/rilldata/rill/cli/pkg/cmdutil"
+	"github.com/rilldata/rill/cli/pkg/config"
+	adminv1 "github.com/rilldata/rill/proto/gen/rill/admin/v1"
+	runtimev1 "github.com/rilldata/rill/proto/gen/rill/runtime/v1"
+	runtimeclient "github.com/rilldata/rill/runtime/client"
+	"github.com/spf13/cobra"
+)
+
+func LogsCmd(cfg *config.Config) *cobra.Command {
+	var name, path string
+	var follow bool
+	var tail int
+
+	logsCmd := &cobra.Command{
+		Use:   "logs [<project-name>]",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "Show project logs",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			client, err := cmdutil.Client(cfg)
+			if err != nil {
+				return err
+			}
+			defer client.Close()
+
+			if len(args) > 0 {
+				name = args[0]
+			}
+
+			if !cmd.Flags().Changed("project") && len(args) == 0 && cfg.Interactive {
+				name, err = inferProjectName(cmd.Context(), client, cfg.Org, path)
+				if err != nil {
+					return err
+				}
+			}
+
+			proj, err := client.GetProject(context.Background(), &adminv1.GetProjectRequest{
+				OrganizationName: cfg.Org,
+				Name:             name,
+			})
+			if err != nil {
+				return err
+			}
+
+			depl := proj.ProdDeployment
+			if depl == nil {
+				return nil
+			}
```

Comment:
Maybe return an error instead, something like "Project %q is not currently deployed"